specification/network/resource-manager/Microsoft.Network/stable/2020-07-01/serviceTags.json

```diff
@@ -163,6 +163,11 @@
           },
           "readOnly": true,
           "description": "The list of IP address prefixes."
+        },
+        "state": {
```

Comment:
Is it an enumable property? If so, you should set it in enum type.

Reply:
This is not an enum.